openfecli/cli.py

```diff
@@ -34,11 +36,13 @@ def get_installed_plugins(self):
 @click.command(cls=OpenFECLI, name="openfe", help=_MAIN_HELP,
                context_settings=CONTEXT_SETTINGS)
 @click.version_option(version=openfecli.__version__)
-def main():
-    # currently empty: we can add options at the openfe level (as opposed to
-    # subcommands) by adding click options here. Subcommand runs after this
-    # is the processed.
-    pass
+@click.option('--log', type=click.Path(exists=True, readable=True),
+              help="logging configuration file")
```

Comment:
I'd also like `--log=True` or something rather than having to mess with files.

Reply:
What would `--log=True` mean? What level of logging are we doing?

How about I allow `DEBUG`, `INFO`, `WARNING`, `ERROR`, or `CRITICAL` (actually, I would allow any capitalization of those, and if you really wanted, I could allow their numeric equivalents as well).

If I do that, what do you want it to apply to? Loggers in `openfe`? Loggers in `openfe` and `gufe`? All loggers? (that would give all debug output from `openmmtools`, `openmm`, `numpy`, etc. -- this is the point of the file-based setup; you can select which loggers you want)